src/cordova/util.js

```diff
@@ -151,12 +152,24 @@ function isCordova (dir) {
     return false;
 }
 
-// Cd to project root dir and return its path. Throw CordovaError if not in a Corodva project.
-function cdProjectRoot () {
-    var projectRoot = convertToRealPathSafe(this.isCordova());
+/**
+ * Returns the project root directory path.
+ *
+ * Throws a CordovaError if not in a Cordova project.
+ */
+function getProjectRoot () {
+    const projectRoot = convertToRealPathSafe(this.isCordova());
+
     if (!projectRoot) {
         throw new CordovaError('Current working directory is not a Cordova-based project.');
     }
+
+    return projectRoot;
+}
+
+// Cd to project root dir and return its path. Throw CordovaError if not in a Corodva project.
+function cdProjectRoot () {
+    const projectRoot = this.getProjectRoot();
```

Comment:
I can't do a proper review right now, but I think the usage of `this` here does hurt more than it helps. I know this follows the style of the rest of `util` but I'd avoid it in new code.

Reply:
I agree, but removing it causes a bunch of tests to fail (I suspect due to improper stubbing)

Reply:
Bummer. I already had a feeling that this could be the reason. I agree that it's better not having to touch the tests then.